samples/basic/another-remote-module/src/dev/App.tsx

```diff
@@ -0,0 +1,5 @@
+import { useAppRouter } from "@basic/shell";
+
+export function App() {
```

Comment:
That feels so weird to look at. Should useAppRouter simply be a component instead? The idea of hook is that it allows the usage of lifecycle methods without having to return JSX. However in this instance, the hook is returning JSX anyway.

Reply:
Oh, yeah, I'll see what I can do.

Reply:
Done, it's now a `AppRouter` component.